### PR TITLE
Fixes fee calculation

### PR DESCRIPTION
**Pull request by uF4No**
Adds gas fee calculation for paymaster transaction after reviewing with @StanislavBreadless .

### CODE REVIEW

File: deploy/use-paymaster.ts [hunk 1]
```diff
@@ -35,17 +35,39 @@ export default async function (hre: HardhatRuntimeEnvironment) {
...
+
+  const fee = gasPrice.mul(gasLimit.toString());
+
   // Encoding the "ApprovalBased" paymaster flow's input
   const paymasterParams = utils.getPaymasterParams(PAYMASTER_ADDRESS, {
     type: 'ApprovalBased',
     token: TOKEN_ADDRESS,
-    minimalAllowance: ethers.BigNumber.from(1),
+    // set minimalAllowance as calculated fee
+    minimalAllowance: fee,
```
Comment (StanislavBreadless) [inline deploy/use-paymaster.ts:58]: `minimalAllowance` is the minimal allowance of the ERC20 token that the user should provide to the paymaster. For the testnet paymaster (the one that is provided by the Matter Labs on the testnet) indeed requires the 1-1 allowance between ETH and ERC20 tokens.

In this particular tutorial, however, the paymaster accepts just 1 unit of the token to pay for any fee, so `ethers.BigNumber.from(1)` should work here

Comment (uF4No) [reply deploy/use-paymaster.ts:58]: cool, thanks for the explanation. I'll include that as a comment.

Comment (uF4No) [reply deploy/use-paymaster.ts]: fixed in 5539d82b2ab0a43972745f063077a259610c9ef1

**Review by StanislavBreadless — APPROVED** [review]

File: deploy/use-paymaster.ts [hunk 2]
```diff
@@ -3,13 +3,13 @@ import * as ethers from 'ethers';
 import { HardhatRuntimeEnvironment } from 'hardhat/types';
 
 // Put the address of the deployed paymaster here
-const PAYMASTER_ADDRESS = '';
+const PAYMASTER_ADDRESS = '<PAYMASTER_ADDRESS>';
 
 // Put the address of the ERC20 token here:
-const TOKEN_ADDRESS = '';
+const TOKEN_ADDRESS = '<TOKEN_ADDRESS';
```
Comment (githubdoramon) [inline deploy/use-paymaster.ts:9]: missing final >

File: package.json [hunk 3]
```diff
@@ -15,6 +15,6 @@
     "@openzeppelin/contracts": "^4.7.3",
     "@openzeppelin/contracts-upgradeable": "^4.7.3",
     "hardhat": "^2.10.2",
-    "zksync-web3": "^0.11.0"
+    "zksync-web3": "^0.11.1"
```
Comment (githubdoramon) [inline package.json:18]: Not for now.. But still think we should just use the "latest" tag

**Pull request merged**